third_party/move/move-model/src/builder/builtins.rs

```diff
@@ -195,6 +398,7 @@ pub(crate) fn declare_builtins(trans: &mut ModelBuilder<'_>) {
                 loc: loc.clone(),
                 oper: Operation::MaxU8,
                 type_params: vec![],
+                type_param_constraints: BTreeMap::default(),
```

Comment:
Actually, you could group similar operations into loops over a literal tuples down here, too, and make the code much more compact and simpler to change in the future.

Reply:
Lets not overdo this. Having this declarations one-by-one is also useful. Moreover, you are asking here to refactor existing code which I have not really touched as part of this PR.

Reply:
Just a thought.